### ISSUE TITLE
Refactor preview: not all checkboxes should be tabstops

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#143736]
Focus on the refactor preview view.
Press tab (wanting to focus on the apply / discard)
Instead focus is moved on all the checkboxes. You have to press tab a lot 🐛 

I suggest to use the same strategy as the breakpoints view.
Tab does not move focus to the checkboxes.
And make sure that your role of your tree items is checkbox (`TreeAccessibilityProvider` has this built in to choose roles)
